deliverables/README.md

```diff
@@ -2,13 +2,25 @@
 #### Required
 ##### Questions
 - What is a framework?
+
+A web framework is a code library that provides functionalities in the code or through extensions that perform common operations required to run web apps. Frameworks are built by reusing code for common HTTP operations and structuring projects so developers can more quickly build and maintain the applications.
+
 - What's the difference between a programming language and a framework?
+
+A programming language is a superset of scripting language and includes anything you yourself write that makes a computer do something. Frameworks are templates of pre-written stuff you can re-use that come with a pre-imposed structure and philosophy of how to approach a domain of problems.
+
 - What’s the difference between a library and a framework?
+
+Libraries are just a straightforward collection of utility classes/methods your code calls upon to get some functionality. In a framework, unlike in libraries, the overall program's "flow of control" is not dictated by the caller, but by the framework, ie a framework has a default behavior referred to as "inversion of control". A framework provides you with the necessary means to extend default behaviours and is non-modifiable by the caller. Simply, your code links in libraries and calls the library whereas the inverted notion is true of a framework; your code is plugged into the framework.
```

Comment:
I didn't copy paste verbatim. I tried to reword, but I do appreciate the reminder to cite my sources. The reason this explanation so closely hugs the Wiki page is it is the best explanation I could find (that made the most sense to me) to describe the difference, especially with that last sentence beginning with "Simply..". What do you think? Do those four characteristics encompass the difference? Is there anything missing? Is there a better way to think of it? Thank you.